examples/usecases/ranking_with_multitask_learning.ipynb

```diff
@@ -50,7 +50,7 @@
     "\n",
```

Comment:
<p>this example link is not correct: For more information about how the schema API works you can check this <a href="https://github.com/NVIDIA-Merlin/models/blob/main/merlin/datasets/entertainment/tenrec_video/schema.pbtxt" target="_blank">example</a>.</p><p> </p><p>I think you wanted to link to this example: https://github.com/NVIDIA-Merlin/models/blob/main/examples/02-Merlin-Models-and-NVTabular-integration.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Thanks. Updated</p>